src/collections/_documentation/apm/distributed-tracing.md

```diff
@@ -0,0 +1,352 @@
+```
+
+**Adding More Spans to the Transaction**
+
+The next example contains the implementation of the hypothetical `process_item` function called from the code snippet in the previous section. Our SDK can determine if there is a current open `transaction` and add all newly created [spans]({%- link _documentation/apm/apm-glossary.md -%}#span) as child operations to the `transaction`. Keep in mind that each individual span also needs to be manually finished; otherwise, spans will not show up in the `transaction`.
+
+You can choose the value of `op` and `description`.
+
+```python
+import sentry_sdk
+
+def process_item(item):
+    
+  # omitted code...
+  with sentry_sdk.start_span(op="http", description="GET /") as span:
+      response = my_custom_http_library.request("GET", "/")
+      span.set_tag("http.status_code", response.status_code)
+      span.set_data("http.foobarsessionid", get_foobar_sessionid())
+```
+
+### JavaScript
+
+To access our tracing features, you will need to install our APM integration:
+
+```bash
+$ npm install @sentry/browser
+$ npm install @sentry/apm
+```
+
+**Sending Traces**
+
+Tracing resides in the `@sentry/apm` package. You can add it to your `Sentry.init` call:
+
+```javascript
+import * as Sentry from '@sentry/browser';
+import { Integrations as ApmIntegrations } from '@sentry/apm';
+
+Sentry.init({
+  dsn: '___PUBLIC_DSN___',
+  integrations: [
+    new ApmIntegrations.Tracing(),
+  ],
+});
+```
+
+You can pass a lot of different options to tracing, but it comes with reasonable defaults out of the box.
+[Spans]({%- link _documentation/apm/apm-glossary.md -%}#span) are instrumented for the following operations within a [transaction]({%- link _documentation/apm/apm-glossary.md -%}#transaction):
+
+- A [transaction]({%- link _documentation/apm/apm-glossary.md -%}#transaction) for every page load
+- All XHR/fetch requests as spans
+
+**Using Tracing Integration for Manual Instrumentation**
+
+Tracing will create a [transaction]({%- link _documentation/apm/apm-glossary.md -%}#transaction) on page load by default; all [spans]({%- link _documentation/apm/apm-glossary.md -%}#span) that are created will be attached to it. Also, tracing will finish the transaction after the default timeout of 500ms of inactivity. The page is considered inactive if there are no pending XHR/fetch requests. If you want to extend the transaction's lifetime beyond 500ms, you can do so by adding more spans to the transaction. The following is an example of how you could profile a React component:
+
+```javascript
+// This line starts an activity (and creates a span).
+// As long as you don't pop the activity, the transaction will not be finished and therefore not sent to Sentry.
+// If you do not want to create a span out of an activity, just don't provide the second arg.
+const activity = ApmIntegrations.Tracing.pushActivity(displayName, {
+        data: {},
+        op: 'react',
+        description: `<${displayName}>`,
```

Comment:
@rhcarvalho do we want to add something about `autoPopAfter`? If so, will you provide the code example?

Reply:
@kamilogorek @HazAT